.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "asf4"]
 	path = asf4
-	url = https://github.com/adafruit/asf4
+	url = https://github.com/jamesadevine/asf4
```

Comment:
merge under lancaster-university or keep using adafruit.

Reply:
can't merge under adafruit

Reply:
Will move to lancaster